src/model/enhance-interactive-atom-elements.ts

```diff
@@ -8,12 +8,12 @@ const enhance = (elements: CAPIElement[]): CAPIElement[] => {
 		if (
 			element._type ===
 			'model.dotcomrendering.pageElements.InteractiveAtomBlockElement'
-		)
+		) {
 			element.html = element.html
-				? // Allow iframes, this is for youtube embeds in interactives, etc
-				  sanitiseHTML(element.html, { ADD_TAGS: ['iframe'] })
+				? sanitiseHTML(element.html, { ADD_TAGS: ['iframe', 'script'] })
 				: element.html;
-		return element;
+			return element;
```

Comment:
Would having the `return element` inside the `if` block mean that all non `InteractiveAtomBlockElement` are set to undefined?

Reply:
Great spot! Thanks. It shows why always wrapping an `if` body in braces is a good idea - avoids this kind of mistake.